cellxgene_schema_cli/cellxgene_schema/matrix_utils.py

```diff
@@ -0,0 +1,135 @@
+from typing import Union
+
+import anndata as ad
+import numpy as np
+from anndata.compat import DaskArray
+from dask.array import map_blocks
+from scipy import sparse
+
+SPARSE_MATRIX_TYPES = {"csr", "csc", "coo"}
+SUPPORTED_SPARSE_MATRIX_TYPES = {"csr"}
+
+
+def get_matrix_format(matrix: DaskArray) -> str:
```

Comment:
i refactored some of the matrix util functions from `utils.py` into `matrix_utils.py`. just seemed a bit cleaner this way and easier to find helper methods.

Reply:
I like this refactor, but [data portal imports get_matrix_format](https://github.com/chanzuckerberg/single-cell-data-portal/blob/e30a8593cb1a785dc4d71b5f81062f186cafacd0/backend/layers/processing/utils/cxg_generation_utils.py#L8) so this'll break it.

Maybe lets mark the other as deprecated, keep both for now, and then make the switch later? and then delete the utils one

Reply:
for safety, let's do that for any other functions that got moved over